core/src/main/java/hudson/cli/handlers/JobOptionHandler.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * The MIT License
+ *
+ * Copyright (c) 2004-2009, Sun Microsystems, Inc.
```

Comment:
I didn't bother doing that since nobody else has when they worked on the source file...  Doesn't matter to me though.